en/docs/get-started/about-this-release.md

```diff
@@ -2,18 +2,16 @@
 
 ### What is new in this release
 
-**WSO2 Identity Server 5.10.0** is the **latest WSO2 Identity Server release** and is the successor of WSO2 Identity Server 5.9.0. It contains the following new features and enhancements:
+**WSO2 Identity Server 5.11.0** is the **latest WSO2 Identity Server release** and is the successor of WSO2 Identity Server 5.10.0. It contains the following new features and enhancements:
 
 <ul>
-    <li><b><a href="../../learn/passwordless-authentication-using-fido2">Passwordless authentication support</a></b></li>
-    <li><b><a href="../../learn/user-portal">An improved User Portal</a></b></li>
-    <li><b><a href="../../develop/application-rest-api">New RESTful APIs for user self-services and server management</a></b></li>
-    <li><b><a href="../../develop/scope-based-authorization-for-internal-rest-apis">Scope based authorization for internal REST APIs</a></b></li>
-    <li><b>Introduce a unique user identifier across the system
-</b><br>
-Previously, the username of the user was considered as an immutable attribute across the system which had multiple limitations. Therefore a new unique user identifier has been introduced along with a new set of user core APIs.
-Also new user store managers have been introduced for all the types such as JDBC, Read-Only LDAP, Read-Write LDAP, and Active directory which can be compatible with the new unique user identifier implementation. 
-</li>
+    <li><b>New react based console application(BETA) with developer and administrator views to manage and maintain the features offered by the Identity Server.</b></li>
+    <li><b>Enhanced My Account application for users to manage their account-related preferences.</b></li>
```

Comment:
Let say in 5.10.0 it was named as `User Portal`

Reply:
and lets link it to `../../learn/my-account`